app/models.py

```diff
@@ -743,6 +758,20 @@ def is_parent_of(self, other):
             other = other.parent
         return False
 
+    def get_users_with_permission(self):
+        service_users = self.users
+        users_with_permission = [str(service_user.user_id) for service_user in service_users]
+
+        return users_with_permission
+
+    users = db.relationship(
```

Comment:
I'd move this up to where the previous relationship was defined. I think we normally try to keep field definitions above method ones.